src/accessibilityStatement/AccessibilityStatement.module.css

```diff
@@ -0,0 +1,53 @@
+.pageWrapper {
+    min-height: 100vh;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+}
+
+.container {
+    margin: 20px 10px 0;
+    width: auto;
+    flex: 1 0 auto;
+}
+
+.innerWrapper {
+    width: auto;
+    margin-left: 10px;
+    margin-bottom: 50px;
+}
+
+
+.innerWrapper p {
+    margin: 16px 0;
+}
+
+.innerWrapper h1 {
+    font-size: var(--fontsize-h-3);
+}
+
+.innerWrapper h2 {
+    margin-top: 25px;
+    font-size: var(--fontsize-h-4);
+}
+
+@media (min-width: 800px) {
```

Comment:
There are different breakpoints in the app, but this is the only one of `800px`. The closes value currently in use would be `768px`. I don't know if it makes a huge difference for this case.

Reply:
I'll change this to `768` to keep things consistent.